pkg/safeguards/helpers.go

```diff
@@ -7,16 +7,15 @@ import (
 	"os"
 	"path/filepath"
 
+	c "github.com/Azure/draft/pkg/safeguards/types"
```

Comment:
can you use a more descriptive alias?

Reply:
This might conflict with the changes [here](https://github.com/Azure/draft/pull/351/files#diff-12c3b1e87a671e42b59caf0efb055cbc20f3112c5613bc4be98fc40298d84731R19)

Reply:
I agree, changed to sgTypes.